index.html

```diff
@@ -4,13 +4,148 @@
+                src="images/nt-basket-icon.svg"
+                alt="" />
+
+              <img class="nt-left-icon" src="images/nt-user-icon.svg" alt="" />
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="nt-hamburger-menu-2">
+        <div class="navbar_items">
+          <ul class="nav-menu">
+            <li class="nav-link">Shop</li>
+            <li class="nav-item">On Sale</li>
+            <li class="nav-item">New Arrivals</li>
+            <li class="nav-item">Brands</li>
+          </ul>
+        </div>
+      </div>
+      <!-- nursahtuncel-OC-4 hero section end-->
+        
+    </header>
+    <main>
+       <!--hero section burada başlıyor-->
+       <section class="hero-section">
+        <div class="hero-section-container">
+          <div class="hero-section-image-container">
+            <img
+              src="images/hero-section/models.png"
+              alt="hero section background image"
+              class="hero-section-background-image"
+            />
+            <img
+              src="images/hero-section/image1.png"
+              alt="Star Icon"
+              class="star-left"
+            />
+            <img
+              src="images/hero-section/image1.png"
+              alt="Star Icon"
+              class="star-right"
+            />
+          </div>
+          <div class="hero-sectin-info">
+            <h1>FIND CLOTHES THAT MATCHES YOUR STYLE</h1>
+            <p>
+              Browse through our diverse range of meticulously crafted garments,
+              designed to bring out your individuality and cater to your sense
+              of style.
+            </p>
+            <button class="hero-section-button">Shop Now</button>
+            <div class="hero-section-stats">
+              <div>
+                <h2>200+</h2>
+                <p>International Brands</p>
+              </div>
+              <div class="vertical-line"></div>
+              <div>
+                <h2>2,000+</h2>
+                <p>High-Quality Products</p>
+              </div>
+              <div class="vertical-line line-right"></div>
+              <div>
+                <h2>30,000+</h2>
+                <p>Happy Customers</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!--hero section burada bitiyor-->
+     
+         
+             
+       <!-- BeyzanurMete-OC-20 Implement the Newsletter  section start -->
+       <div class="newsletter-section">
+        <div class="newsletter-title">
+            STAY UP TO DATE ABOUT <br> OUR LATEST OFFERS
+        </div>
+        <div class="newsletter-form">
+            
+            <img src="images/Frame.png" alt="Email Icon" class="input-icon">
+            <input type="email" class="newsletter-input" placeholder="Enter your email address" />
+          
+            <button class="newsletter-button">Subscribe to Newsletter</button>
+        </div>
```

Comment:
Bu section'in kapama </div> tag'ini unutmusuz. 

Footer altinda alalim bu section'i. Aciklamasini css icerisinde yaptim.